zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Note.java

```diff
@@ -650,6 +652,40 @@ public void setInfo(Map<String, Object> info) {
     this.info = info;
   }
 
+  public Interpreter getRepl(String name) {
+    return factory.getInterpreter(getId(), name);
+  }
+
+  public Map<String, Object> getEditorSetting(String replName) {
```

Comment:
I think this method is more make sense to be placed in InterpreterFactory, while editor setting is related interpreter and interpreter setting, not each note. What do you think?